core/src/main/resources/lib/form/number.jelly

```diff
@@ -57,13 +57,19 @@ THE SOFTWARE.
     </st:attribute>
     <st:attribute name="max"> <![CDATA[
       The maximum of the @value. This becomes the @max of the <input> tag.
-      This will work only @clazz is 'number', 'number-required', 'non-negative-number-required',
+      This will work only if @clazz is 'number', 'number-required', 'non-negative-number-required',
         'positive-number', 'positive-number-required'.
       If specified, the @value should be less than this value, or errors will be rendered under the text field.
       If this value contains non-digit characters, it will not work.
       If @min is specified and @min is greater than this value, both @min and @max will not work.
        ]]>
     </st:attribute>
+    <st:attribute name="step">
+      The step size of the @value. The default is 1.
+      This will work only if @clazz is 'number', 'number-required', 'non-negative-number-required',
+      'positive-number', 'positive-number-required'.
```

Comment:
Why would that be the case? `min` and `max` are specifically supported by `registerMinMaxValidator`. `step` is only used by the browser's basic number spinner control.